schema/google/showcase/v1beta1/messaging.proto

```diff
@@ -283,10 +283,10 @@ message ListRoomsResponse {
 message Blurb {
   option (google.api.resource) = {
     type: "showcase.googleapis.com/Blurb"
-    pattern: "users/{user}/profile/blurbs/legacy/{legacy_user}~{blurb}"
-    pattern: "users/{user}/profile/blurbs/{blurb}"
+    pattern: "users/{user}/blurbs/{blurb}"
     pattern: "rooms/{room}/blurbs/{blurb}"
-    pattern: "rooms/{room}/blurbs/legacy/{legacy_room}.{blurb}"
+    pattern: "rooms/{room}/legacy_room/{legacy_room}/blurbs/{blurb}"
```

Comment:
Why are these modified/added resource names needed? (I'll need to understand resource names more deeply in general.)